demisto_sdk/__main__.py

```diff
@@ -131,7 +131,8 @@ def main(config, version, release_notes):
             __version__ = get_distribution('demisto-sdk').version
         except DistributionNotFound:
             __version__ = 'dev'
-            print_warning('Cound not find the version of the demisto-sdk. This usually happens when running in a development environment.')
+            print_warning(
```

Comment:
These look like an automated code formatting changes. To make `git blame` easier, it's recommended to avoid them and discard the changes (if something breaks devs will see this PR as the last change instead of the real PR where these lines were introduced). Your call.